modules/container-configbaker/Dockerfile

```diff
@@ -10,7 +10,10 @@ ARG SOLR_VERSION
 FROM solr:${SOLR_VERSION} AS solr
 
 # Let's build us a baker
-FROM alpine:3
+# WARNING:
+# Do not upgrade the tag to :3 or :3.19 until https://pkgs.alpinelinux.org/package/v3.19/main/x86_64/c-ares is at v1.26.0+!
```

Comment:
I'm confused, won't https://pkgs.alpinelinux.org/package/v3.19/main/x86_64/c-ares (Alpine 3.19) always be where it is, a c-ares  1.24.0-r1?
![Screenshot 2024-03-22 at 11 24 17 AM](https://github.com/IQSS/dataverse/assets/21006/4c67d67d-87c9-4dfb-8385-49a49f6b1be0)

Or are you saying I can check back in a year and Alpine 3.19 will have a newer version of c-ares?

Reply:
I'm not sure, but I would think so. I didn't lookup their policy about merging in edge packages into a main branch.

Reply:
Wait... I see they do backports! https://git.alpinelinux.org/aports/commit/?id=6edd0d6d61c7f8b8a37e84333b37aa02e6507bfb

Reply:
Ok, thanks. I think the comment is fine. Resolving.